management/src/main/java/org/kaazing/gateway/management/snmp/transport/ManagementUdpTransport.java

```diff
@@ -61,7 +61,8 @@ public void sendMessage(final Address address, final byte[] message) throws IOEx
                 URI connectURI = new URI("udp://" + udpAddress.getInetAddress().getHostAddress() + ":" + udpAddress.getPort());
```

Comment:
Adding a change for this with my latest commit (will be pushed to my remote branch in 5 minutes).